setup.py

```diff
@@ -18,6 +18,6 @@
       license='Apache 2.0',
       description='Cloudstate Python Support Library',
       packages=find_packages(exclude=['tests', 'shoppingcart']),
-      long_description=open('README.md', 'r').read(),
+      long_description=open('README.txt', 'r').read(),
```

Comment:
we could rename README.txt to DESCRIPTION.md, so that the *.txt is in line with the content and the README.md still offers information about the repository in github.

Reply:
Renamed to Description.md